### DISCUSSION TITLE
Duplicate Title in the Readme

### DISCUSSION THREAD

**Discussion by jfmartinz** [gautamkrishnar/blog-post-workflow#256] [Q&A]
I tested it out, just a random stuff in the blog, just to test it if its working, and I got this when I published the blog in medium. 

![image](https://github.com/gautamkrishnar/blog-post-workflow/assets/129386460/3d4f8336-6381-4b90-9f56-04634df71b40)

The problem is, I got two title even though I just add one blog post from medium. Not sure, the problem is. I run the yaml file manually and its passing. Can anyone help me whats the issue with this yaml file? I think its because of the  `feed_list`?

Yaml file:

```shell
name: Latest blog post workflow
on:
  schedule: # Run workflow automatically
    - cron: '0 * * * *' # Runs every hour, on the hour
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the GitHub Actions Workflow page directly
permissions:
  contents: write # To write the generated contents to the readme

jobs:
  update-readme-with-blog:
    name: Update this repo's README with latest blog posts
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4
      - name: Pull in dev.to posts
        uses: gautamkrishnar/blog-post-workflow@v1
        with:
          feed_list: "https://medium.com/feed/@jfmartinz, https://medium.com/feed/@jfmartinz"
```

**Comment by gautamkrishnar** [comment]
@jfmartinz At some point of time medium might've glitched and sent back those lists as feed. Thats why it is showing in the readme. It looks like you don't have any bog posts. 

So just delete those lines from readme and you are good to go.  Except the 2 duplicate links there is nothing wrong with that yaml.

**Comment by jfmartinz** [comment]
I dleted already the blog post since that is just for testing purposes how would the readmme look like when I actually post a blog.

**Comment by jfmartinz** [comment]
Ill delete the two bullet points?
